schemas/_common-schema.json

```diff
@@ -26,13 +27,16 @@
         "id": {
           "type": "string",
           "description": "UUID of the application",
-          "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
+          "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$",
+          "readOnly": true
         }
-      }
+      },
+      "readOnly": true
```

Comment:
hum, the current change don't impact the format validation, so I'd be in favor of not mixing the concerns.

FMU, If we introduce `additionalProperties: false`, we would need to bump format version each time we add a new field, so it may worth to discuss it a bit first.

Reply:
I didn't think about that indeed, so yeah you're right we shouldn't add the `additionalProperties`